examples/ct/ct_handlers_test.go

```diff
@@ -168,20 +177,23 @@ func TestGetHandlersRejectPost(t *testing.T) {
 		s := httptest.NewServer(hp.handler)
 		defer s.Close()
 
-		resp, err := http.Post(s.URL + "/ct/v1/" + hp.path, "application/json", nil)
+		resp, err := http.Post(s.URL+"/ct/v1/"+hp.path, "application/json", nil)
 
 		if err != nil {
 			t.Fatal(err)
 		}
 
-		assert.Equal(t, http.StatusMethodNotAllowed, resp.StatusCode, "Wrong status code for POST to GET handler")
+		if expected, got := http.StatusMethodNotAllowed, resp.StatusCode; expected != got {
+			t.Fatalf("Wrong status code for POST to GET handler, expected %v, got %v", expected, got)
+		}
 	}
-
-	client.AssertExpectations(t)
 }
 
 func TestPostHandlersRejectEmptyJson(t *testing.T) {
-	client := new(trillian.MockTrillianLogClient)
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
```

Comment:
Yup, `.Finish()` does all of that, and it seems to be stricter in some cases too - doing an `EXPECT()...blah().Returns()` will match once only - you need to explicitly add and `.AnyTimes()` (or whatever) otherwise.